package.json

```diff
@@ -35,6 +37,7 @@
     "@polkadot/react-identicon": "2.10.1",
     "@react-spring/web": "9.6.1",
     "@use-gesture/react": "10.2.23",
+    "commander": "9.5.0",
```

Comment:
Do we need that?

Reply:
This is needed till the below fix is merged.
https://github.com/paritytech/zombienet/pull/703

Reply:
Now fixed upstream